src/main/java/com/conveyal/datatools/manager/jobs/NotifyUsersForSubscriptionJob.java

```diff
@@ -27,32 +27,19 @@
     private static final String APPLICATION_NAME = DataManager.getConfigPropertyAsText("application.title");
     private static final String APPLICATION_URL = DataManager.getConfigPropertyAsText("application.public_url");
 
-    private NotifyUsersForSubscriptionJob(String subscriptionType, String target, String message) {
+    public NotifyUsersForSubscriptionJob(String subscriptionType, String target, String message) {
+        super();
```

Comment:
Please revert due to reasons list in https://github.com/ibi-group/datatools-server/pull/348#discussion_r512341045.

Reply:
Updated.